boards/riscv/longan_nano/doc/index.rst

```diff
@@ -0,0 +1,62 @@
+.. _longan_nano:
+
+Sipeed Longan Nano
+##################
+
+Overview
+********
+
+The Sipeed Longan Nano and Longan Nano Lite is an simple and tiny development board with
+an GigaDevice GD32VF103 SoC that based on N200 RISC-V IP core by Nuclie system.
+More information can be found on
+`Sipeed's website <https://longan.sipeed.com/en/>`_ and
+`Nuclei's website <https://www.nucleisys.com/download.php>`_.
```

Comment:
nit: add links to the GD32VF103 SoC datasheet and ref manual, similar to https://docs.zephyrproject.org/latest/boards/arm/gd32f450i_eval/doc/index.html

Reply:
Add these links.
- Sipeed Longan website https://longan.sipeed.com/en/
- GD32VF103 datasheet https://www.gigadevice.com/datasheet/gd32vf103xxxx-datasheet/
- GD32VF103 user manual http://www.gd32mcu.com/data/documents/shujushouce/GD32VF103_User_Manual_EN_V1.2.pdf
- Nuclei website https://www.nucleisys.com/download.php
- Nuclei Bumblebee core documents https://github.com/nucleisys/Bumblebee_Core_Doc
- Nuclei ISA Spec https://doc.nucleisys.com/nuclei_spec